c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -136,6 +136,12 @@ Version 1.6
 13. Remove your Apache configuration file, probably in
     `/var/www/vhost/{{project}}/conf`. Now it will be automatically created.
 
+14. Notes on database changes:
+    * We add some tables and do some changes for GeoMapFish 2.0
+    * We move the user table in the static schema than all the data that
+      can be modified by the webapplication are in the same schema.
+      => we remove the password replication
```

Comment:
Or maybe:
As a result the passwords no longer need to be synchronised among the various instances of the application and the password replication tool has been removed.